net-dns/dnsdist/metadata.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person">
+		<email>bgo@9dt.de</email>
+		<name>Vladimir Datsevich</name>
+	</maintainer>
+	<maintainer type="project">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<longdescription lang="en">
+	</longdescription>
+	<use>
+		<flag name="clang">Use Clang compiler instead of GCC</flag>
+		<flag name="dnscrypt">Enable DNSCRypt support</flag>
+		<flag name="readline">Enable support for libreadline, a GNU line-editing library that almost everyone wants</flag>
+		<flag name="libedit">Use the libedit library (replacement for readline)</flag>
+		<flag name="sodium">Use dev-libs/libsodium for cryptography</flag>
```

Comment:
I'll quote:

> dnsdist, when compiled with --enable-dnscrypt, can be used as a DNSCrypt server, uncurving queries before forwarding them to downstream servers and curving responses back.

whereas sodium:

> It can optionally use libsodium for encrypted communications with its client

They are not the same thing.

Woud it be better to rename sodium to crypt then?